plugins/playstation/src/playstation-plugin.vala

```diff
@@ -0,0 +1,32 @@
+// This file is part of GNOME Games. License: GPLv3
+
+private class Games.PlayStation : Object, Plugin {
+	private const string FINGERPRINT_PREFIX = "playstation";
+	private const string MIME_TYPE = "application/x-cue";
```

Comment:
Much too far. But the work-around is to add a miner in tracker as done here for a number of types:
https://git.gnome.org/browse/tracker/tree/src/tracker-extract

You'd filter by mime-type (the one above) and add a property to the extracted data that would tag it as a PSX game. This code can then also be used to differentiate other types of images. There might also be something in tracker to mention that this is just a "playlist" and say that a separate file is also necessary to make this work.